kagami/server.py

```diff
@@ -15,7 +15,7 @@
 @asynccontextmanager
 async def lifespan(app: FastAPI):
     global supervisor
-    supervisor = Supervisor  # TODO load the supervisor
+    supervisor = await Supervisor.load()
```

Comment:
DOES IT EXISTS??????

https://github.com/ProjectSoftoid/Kagami/blob/f6a820731d68321c0de264a8b7ccb9a4932724ac/kagami/core/supervisor.py#L46-L49

Reply:
This function will be implement soon. So please comment this function here and don't do changes until this function is fully implemented.

Reply:
And please setup pre-commit hooks correctly. I remember i have told you in https://github.com/ProjectSoftoid/Kagami/pull/17#discussion_r1860047655 and you didn't resolve this conversation. Please remember to check all the reviews and resolve all the conversation before. Thank you.

Reply:
We don't have CI right now, so please enable pre-commit hooks. If you setup ruff and mypy check correctly, they will tell you this function is not exist at all and refuse you to create commit.

Reply:
> We don't have CI right now, so please enable pre-commit hooks. If you setup ruff and mypy check correctly, they will tell you this function is not exist at all and refuse you to create commit.

Received, I'll use it before the next commit